std/stdio.d

```diff
@@ -462,12 +471,12 @@ Assigns a file to another. The target of the assignment gets detached
 from whatever file it was attached to, and attaches itself to the new
 file.
  */
-    void opAssign(File rhs) @safe
-    {
-        import std.algorithm : swap;
-
-        swap(this, rhs);
-    }
+//    void opAssign(File rhs) @safe
+//    {
+//        import std.algorithm : swap;
+//
+//        swap(this, rhs);
+//    }
 
```

Comment:
If `opAssign()` is no longer needed, just remove it altogether with its comment.